images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -14,6 +14,13 @@ ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR \
     LC_ALL="C.UTF-8" \
     ORYX_PATHS="/opt/oryx:/opt/nodejs/lts/bin:/opt/python/latest/bin:/opt/yarn/stable/bin"
     
+# stretch was removed from security.debian.org and deb.debian.org, so update the sources to point to the archived mirror
```

Comment:
Could you please clarify how you see this changing in the future? For example, can we assume that we'll be able to remove all references to Stretch from our code (I believe we only keep them while our partners are deploying changes) before the archived version disappears?

Reply:
I think the archive would remain for about 2-3 years, and then be expired (I think I saw this info somewhere in Debian distro webpage, but I couldn't find the link for now...). I think we should be stable for now, but imo it would be best for us to move from stretch in the meantime.